crates/ln-dlc-node/src/node/dlc_channel.rs

```diff
@@ -23,34 +23,26 @@ where
     P: Send + Sync,
 {
     #[autometrics]
-    pub async fn propose_dlc_channel(
+    pub fn propose_dlc_channel(
```

Comment:
same as elsewhere, I feel that having it `async` was better,  but we should have *multiple* `spawn_blocking` instead of one (if the whole body would stay like this, then we might as well remove it).